types/src/error.rs

```diff
@@ -67,6 +67,9 @@ pub enum Error {
 	/// Invalid request ID.
 	#[error("Invalid request ID")]
 	InvalidRequestId,
+	/// Unregistered notification method
```

Comment:
Can we extend the docs here a little bit, it's not immediately clear (to me) what "notification" means here, it being a word used for a few different things in the library. Maybe just a note about when this can happen?